.github/workflows/build-dev-release.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish docker image dev release
```

Comment:
note: both release and dev release workflows look exactly the same except `-dev` suffix for the image name. Maybe we should join them and use one more input parameter to choose between dev and regular release